components/playback/controls/player/AddCustomer.vue

```diff
@@ -6,16 +6,54 @@
     </span>
     <div id="uuid" style="top: 40px; left: 50px; position: absolute">
       <InputWidget
-        :title_label="label_uuid"
-        :placeholder="keyplaceholder_encode"
-        :spellcheck="spellchecking"
+        :title_label="'Enter Alphanumeric ID'"
+        :placeholder="'2VSckkBYr2An3dqHEyfRRE'"
+        :spellcheck="false"
         :invalid_text="error_text_uuid"
-        :input_width="entry_width"
-        :disabled="disallow_entry"
+        :input_width="400"
+        :disabled="false"
         :dom_id_suffix="'alphanumeric-id'"
         @update:value="on_update_uuid($event)"
       ></InputWidget>
     </div>
+
+    <div id="apikey" style="top: 140px; left: 50px; position: absolute">
+      <InputWidget
+        :title_label="'Enter API Key(Optional)'"
+        :placeholder="'ba86b8f0-6fdf-4944-87a0-8a491a19490e'"
+        :spellcheck="false"
+        :invalid_text="error_text_api"
+        :input_width="400"
+        :disabled="false"
+        :dom_id_suffix="'apikey-id'"
+        @update:value="on_update_api($event)"
+      ></InputWidget>
+    </div>
+    <div id="nickname" style="top: 240px; left: 50px; position: absolute">
+      <InputWidget
+        :title_label="'Enter ID Nickname'"
+        :placeholder="'Curi Bio Main Account'"
+        :spellcheck="false"
```

Comment:
The "Nickname" is typically something in plain English. Is there a reason you think spellcheck should be disabled for this field?

Reply:
Spellcheck is to verify spelling and grammar checked, if set not set 'false' by default it would be 'true' and there is no UI/UX to show a squiggle text under nickname, which will get applied so its essential to have spellcheck="false" for nickname.

Reply:
per the conversation with Ari on Slack today, please make a note to update `spellcheck` to true (or just remove it since the default is true) as part of your next pull request